app/models/koi_model.rb

```diff
@@ -0,0 +1,6 @@
+class KoiModel < ActiveRecord::Base
```

Comment:
i'm not sure.
maybe it should be a concern rather than a class? and every koi model would include KoiModel ?

Reply:
A concern would probably be more consistent with a modular approach and better for including in apps (otherwise we end up with parallel inheritance paths, which happened with `CrudController` and it's never nice).

I'd still consider name-spacing it, but that's more personal preference than any real need.

Reply:
i've changed the models to use a namespaced concern
`include Koi::Model` to include all koi model functionality